### DISCUSSION TITLE
Hot key  for full expression

### DISCUSSION THREAD

**Discussion by jandiradmin** [Qalculate/qalculate-qt#86] [Ideas]
Is it possible to add a hotkey to copy the expression to the buffer in the format: 2*2=4

**Comment by hanna-kn** [comment]
It is not possible in any released version, but I've now added this as an option to the copy result keyboard shortcut action (https://github.com/Qalculate/qalculate-qt/commit/ce71a948e692f592e27983830a508171222ad9b8).
